esp/services/ws_sql/ws_sqlService.cpp

```diff
@@ -772,8 +770,7 @@ bool CwssqlEx::onGetRelatedIndexes(IEspContext &context, IEspGetRelatedIndexesRe
 {
     try
     {
-        if (!context.validateFeatureAccess(WSSQLACCESS, SecAccess_Read, false))
-            throw MakeStringException(-1, "Failed to execute SQL. Permission denied.");
+        context.ensureFeatureAccess(WSSQLACCESS, SecAccess_Read, -1, "Failed to execute SQL. Permission denied.");
```

Comment:
original error message was wrong. should be "WsSQL::onGetRelatedIndexes: Permission denied."